src/client.ts

```diff
@@ -2242,6 +2215,40 @@ export class StreamChat<
     return false;
   }
 
+  _muteStatus(cid: string) {
+    let muteStatus;
+    this.mutedChannels.forEach(function (mute) {
+      if (mute.channel?.cid === cid) {
+        let muted = true;
+        if (mute.expires) {
+          muted = new Date(mute.expires).getTime() > new Date().getTime();
+        }
+        muteStatus = {
+          muted,
+          createdAt: mute.created_at ? new Date(mute.created_at) : new Date(),
+          expiresAt: mute.expires ? new Date(mute.expires) : null,
+        };
+      }
+    });
+
+    if (muteStatus) {
+      return muteStatus;
+    }
+
+    return {
+      muted: false,
+      createdAt: null,
+      expiresAt: null,
+    };
+  }
+
+  /**
+   * channelMuteStatus - check if the channel is muted or not, can be used after connectUser() is called
+   *
+   * @param cid
+   */
+  channelMuteStatus = (cid: string) => this._muteStatus(cid).muted;
```

Comment:
we have a similar function for `userMuteStatus`, so makes sense to have one for channel as well!!